server/src/internalClusterTest/java/org/elasticsearch/search/functionscore/FunctionScorePluginIT.java

```diff
@@ -146,7 +146,6 @@ private static class LinearMultScoreFunction implements DecayFunction {
 
             @Override
             public double evaluate(double value, double scale) {
-
```

Comment:
Spacing nit: We probably don't need this change.

Reply:
I know, it is just aesthetics :)